pkg/util/mount/nsenter_mount.go

```diff
@@ -170,6 +170,12 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, err
 	}
 
+	// Check the directory exists
+	if _, err = os.Stat(file); os.IsNotExist(err) {
```

Comment:
Kubernetes runs with /var/lib/kubernetes mounted to the same directory in the container so it works.

If you look e.g. into AWS volume plugin, it just calls `mkdir()` there without any nsenter. https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/aws_ebs/aws_util.go#L75